guides/common/modules/con_scap-content.adoc

```diff
@@ -1,18 +1,17 @@
 [id="SCAP_Content_{context}"]
 = SCAP Content
 
-SCAP content is a datastream format containing the configuration and security baseline against which hosts are checked.
-Checklists are described in the extensible checklist configuration description format (XCCDF) and vulnerabilities in the open vulnerability and assessment language (OVAL).
-Checklist items, also known as rules express the desired configuration of a system item.
-For example, you may specify that no one can log in to a host over SSH using the `root` user account.
-Rules can be grouped into one or more profiles, allowing multiple profiles to share a rule.
-SCAP content consists of both rules and profiles.
+SCAP content is a SCAP data-stream file that contains implementation of compliance, configuration, or security baselines.
+A single data stream usually includes multiple XCCDF profiles.
+An XCCDF profile defines an industry standard or custom security standard against which you can evaluate compliance of host configuration in {Project}.
+You can also tailor existing XCCDF profiles according to requirements of your company using _tailoring files_.
+
+In {Project}, you use an XCCDF profile from a SCAP content and, eventually, a tailoring file, to define a _compliance policy_.
+{Project} includes default SCAP contents from SCAP Security Guide provided by the https://www.open-scap.org/[OpenSCAP project].
+You have to load a SCAP content and tailoring file into {ProjectServer} before you can apply it in a compliance policy.
 
-You can either create SCAP content or obtain it from a vendor.
-Supported profiles are provided for {RHEL} in the `scap-security-guide` package.
 ifndef::orcharhino[]
-The creation of SCAP content is outside the scope of this guide, but see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/security_hardening/[{RHEL}{nbsp}8 Security hardening] for information on how to download, deploy, modify, and create your own content.
+For more information on how to download, deploy, modify, and create your own content, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/security_hardening/[_{RHEL}{nbsp}8 Security hardening_].
 endif::[]
 
-The default SCAP content provided with the OpenSCAP components of {Project} depends on the version of {RHEL}.
-On {RHEL} 7, content for both {RHEL} 6 and {RHEL} 7 is installed.
+{Project} supports content of SCAP versions 1.2 and 1.3.
```

Comment:
Or @ekohl ? ^^

Reply:
My initial question was aiming towards "if we merge this; to which branch can we cherry-pick it?". Since the answer is "not even "3.5"; then my questions becomes less relevant. If Foreman nightly + its OpenSCAP plug-in supports both 1.2 and 1.3, then feel free to leave it as is.